src/GrumPHP/Task/NpmScript.php

```diff
@@ -59,7 +61,12 @@ public function run(ContextInterface $context)
             return TaskResult::createSkipped($this, $context);
         }
 
-        $arguments = $this->processBuilder->createArgumentsForCommand('npm');
+        $command = 'npm';
+        if (true === $config['is_run_task']) {
```

Comment:
@bvanderstee 

The `addOptionalArgument()` method is created to decrease the amount of if statements in the tasks. A better use-case in which you don't need the if-statement looks like this:

``
$arguments->addOptionalArgument('run',  $config['is_run_task']);
```